lms/static/js/ccx/view/ccx_schedule.js

```diff
@@ -0,0 +1,394 @@
+var define = window.define || RequireJS.define;  // jshint ignore:line
+
+define(
+  [
+    'backbone',
+    'jquery',
+    'underscore',
+    'gettext',
+    'moment',
+    'js/ccx/collection/schedule_collection',
+    'js/ccx/view/schedule_tree',
+    'js/ccx/view/schedule_right_container',
+    'text!templates/ccx/underscore/ccx_schedule.underscore'
+  ],
+  function (
+    Backbone,
+    $,
+    _,
+    gettext,
+    moment,
+    scheduleCollection,
+    scheduleTree,
+    scheduleRightContainerView,
+    scheduleTemplate
+  ) {
+    'use strict';
+    var ccxScheduleView = Backbone.View.extend({
+
+      initialize: function(options) {
+        this.saveCCXScheduleUrl = options.saveCCXScheduleUrl;
+        this.unSavedChanger = false;
+      },
+
+      render: function() {
+        this.scheduleJson = this.collection.toJSON();
```

Comment:
Instead of dealing only in json, use the collections. They're pretty useful (especially for listening when things change) and you can convert them to json whenever you want.